.github/workflows/ci.yml

```diff
@@ -145,9 +145,17 @@ jobs:
           pnpm run test-browser
           pnpm build-browser-bundle --npmDir npm-dir
 
+  # Sharded because these tests are slow
   test-napi:
     name: Test NAPI
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        component:
+          - oxlint2
+          - parser
+          - playground
```

Comment:
We are spawning too many jobs.

Reply:
The entire purpose of this PR is to spawn more jobs!